upgrade/upgradetest/config.go

```diff
@@ -76,3 +79,40 @@ func SetTimesTo(c *upgrade.Config, fork Fork, upgradeTime time.Time) {
 		c.ApricotPhase1Time = upgradeTime
 	}
 }
+
+// SetTimeOnly sets the upgrade time of only the specified fork to the provided
+// upgradeTime, without affecting any other fork times.
+func SetTimeOnly(c *upgrade.Config, fork Fork, upgradeTime time.Time) {
```

Comment:
This is in the `upgradetest` package but used in production code -- however doing it otherwise results in an cyclic import loop. This seems bad.